### PR TITLE
feat: setup UI endpoint to serve the UI

### LINKED ISSUE #39: setup UI endpoint to serve the UI
PR #37 is adding the UI stuff but not actually serving it.

This is a followup for task #19 : need to redirect the user to the UI endpoint when finished handling the OIDC callback. The redirectURL should contain a handle of some sort that can be used by the UI to fetch the user's bootstrap info and offer it to the wallet via CHAPI.

### PR DESCRIPTION
**Pull request by talwinder50**
closes #39

Signed-off-by: talwinder.kaur <talwinder.kaur@securekey.com>

### CODE REVIEW

File: pkg/restapi/operation/operations.go [hunk 1]
```diff
@@ -359,6 +374,13 @@ func handleAuthError(w http.ResponseWriter, status int, msg string) {
 	}
 }
 
+func (c *Operation) setUserProfile(handle string, r *user.Profile) {
+	c.presDefsLock.Lock()
+	defer c.presDefsLock.Unlock()
+
+	c.userProfileRequest[handle] = r
```
Comment (rolsonquadras) [inline pkg/restapi/operation/operations.go:381]: Shouldn't we set this in database instead of local map ?

Comment (talwinder50) [reply pkg/restapi/operation/operations.go:381]: @llorllale @aholovko what you guys recommend ?

Comment (llorllale) [reply pkg/restapi/operation/operations.go:381]: @talwinder50 should use transient store

File: pkg/restapi/operation/operations.go [hunk 2]
```diff
@@ -342,9 +351,15 @@ func (c *Operation) onboardUser(id string) (*user.Profile, error) {
 	return userProfile, nil
 }
 
-// TODO redirect to the UI: https://github.com/trustbloc/hub-auth/issues/39
-func handleAuthResult(w http.ResponseWriter, r *http.Request, _ *user.Profile) {
-	http.Redirect(w, r, "", http.StatusFound)
+func (c *Operation) handleAuthResult(w http.ResponseWriter, r *http.Request, userProfile *user.Profile) {
+	handle := url.QueryEscape(uuid.New().String())
```
Comment (talwinder50) [inline pkg/restapi/operation/operations.go:355]: Is this handle should be fetched from the UI as described here: https://github.com/trustbloc/hub-auth/issues/39

File: pkg/restapi/operation/operations.go (hunk 1 above)
Comment (talwinder50) [reply pkg/restapi/operation/operations.go]: done

**Comment by talwinder50** [comment]
> ui endpoint handlers not added to router

handleOIDCCallback is calling that handleAuthResult,  it's already registered. Sorry What do you mean by this ?

**Comment by talwinder50** [comment]
i have marshal func added in operation reducing the code cov. Overall project cov is 86%

File: cmd/auth-rest/startcmd/start.go [hunk 3]
```diff
@@ -143,6 +149,7 @@ type authRestParameters struct {
 	tlsParams       *tlsParams
 	oidcParams      *oidcParams
 	bootstrapParams *bootstrapParams
+	staticFiles     string
```
Comment (llorllale) [inline cmd/auth-rest/startcmd/start.go:152]: `staticFiles` is not being assigned a value

Comment (talwinder50) [reply cmd/auth-rest/startcmd/start.go:152]: done

Comment (talwinder50) [reply cmd/auth-rest/startcmd/start.go:152]: done

**Pull request merged**